uint32 pointInImageX
uint32 pointInImageY
---
bool success
---
float32 distance